src/storage.ts

```diff
@@ -25,16 +40,36 @@ const V0_DEDUP_RTV_PREFIXES: ReadonlySet<string> = new Set([
  * https://developers.cloudflare.com/workers/learning/how-the-cache-works#interacting-with-the-cloudflare-cache
  *
  * @param url - to fetch from cache or network.
+ * @param cf - incoming Cloudflare properties.
  * @returns a Response object for the request URL.
  */
-export async function fetchImmutableUrlOrDie(url: string): Promise<Response> {
+export async function fetchImmutableUrlOrDie(
+  url: string,
+  cf?: IncomingCloudflarePropertiesExtended
+): Promise<Response> {
   const response = await fetch(url, {
     cf: {cacheEverything: true, cacheTtl: 31536000},
   });
   if (!response.ok) {
     throw new FetchError(response.status, response.statusText);
   }
-  return response;
+
+  if (!tryBrotli(cf)) {
```

Comment:
This serializes the requests, instead of parallelizing.

Reply:
Fixed 35574b5

Reply:
Fixed 35574b5